declare_clippy_lint/Cargo.toml

```diff
@@ -0,0 +1,10 @@
+[package]
+name = "declare_clippy_lint"
+version = "0.1.89"
```

Comment:
Should be added to `tests/versioncheck.rs`

Reply:
Probably should. I don't think I added it to `clippy_dev` either.